homework/myapp/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from .models import Products
+
+
+class ProductCreateForm(forms.Form):
+    title = forms.CharField(max_length=60, label='Название продукта', widget=forms.TextInput(attrs={'class': 'input__form', 'required': True}))
+    description = forms.CharField(max_length=200, label='Описание продукта', widget=forms.TextInput(attrs={'class': 'input__form', 'required': True}))
+    price = forms.DecimalField(max_digits=10, decimal_places=2, label='Стоимость продукта', widget=forms.NumberInput(attrs={'class': 'input__form', 'required': True}))
+    quantity = forms.IntegerField(label='Количество', widget=forms.NumberInput(attrs={'class': 'input__form', 'required': True}))
+    image = forms.ImageField(label='Добавтье изображение')
+
+    class Meta:
+        model = Products()
+        fields = ['title', 'description', 'price', 'quantity', 'image']
```

Comment:
Можно сделать иначе, как показывал @proDreams на семинаре
```suggestion
class ProductCreateForm(forms.ModelForm):
    class Meta:
        model = Products
        fields = {'title',
                  'description',
                  'price',
                  'quantity',
                  'image'}

        labels = {'title': 'Название продукта',
                  'description': 'Описание продукта',
                  'price': 'Стоимость продукта',
                  'quantity': 'Количество',
                  'image': 'Добавьте изображение'}
```

Reply:
> Можно сделать иначе, как показывал @proDreams на семинаре

Согласен, нет смысла переопределять поля формы, если ничего в них не вносится, например, кастомные стили или стили из bootstrap или дополнительные правила валидации. Всё остальное форма модели возьмёт из описания полей модели.